catroid/src/main/java/org/catrobat/catroid/content/actions/GoToOtherSpritePositionAction.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2016 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.content.actions;
+
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
+
+import org.catrobat.catroid.content.Sprite;
+
+public class GoToOtherSpritePositionAction extends TemporalAction {
+	private Sprite sprite;
+	private Sprite destinationSprite;
+
+	@Override
+	protected void update(float percent) {
+		if (destinationSprite == null) {
+			destinationSprite = this.sprite;
```

Comment:
Please just return, saves the app from executing 4 lines of code that basically do nothing (i know this will never hit performance, but if its not needed you don't need to do it ;D)